weblate/api/views.py

```diff
@@ -951,6 +952,23 @@ def delete_links(self, request, project__slug, slug, project_slug):
         instance.links.remove(project)
         return Response(status=HTTP_204_NO_CONTENT)
 
+    @action(detail=True, methods=["get"], url_path="translations/zip")
```

Comment:
Maybe it would be better to use `/file/` to be consistent with other download endpoints? But I'm not convinced whether it's better...

```suggestion
    @action(detail=True, methods=["get"], url_path="file")
```

Reply:
Okay, let's make it consistent with other downloads:

* call the endpoint file
* accept format in the URL and pass it to download_multi (it is currently not used, but will allow extending in future)